source/qmltoolbox/source/qmltoolbox.cpp

```diff
@@ -11,10 +10,11 @@ namespace
 
 std::string determineDataPath()
 {
-    const cpplocate::ModuleInfo moduleInfo = cpplocate::findModule("qmltoolbox");
-    const std::string moduleInfoPath = moduleInfo.value("dataPath");
+    std::string path = cpplocate::locatePath("data/qmltoolbox", "share/qmltoolbox", reinterpret_cast<void *>(&qmltoolbox::dataPath));
```

Comment:
Seems like we should add a function template to cpplocate that wraps the ```void *``` cast so a user may write ```cpplocate::locatePath("data/qmltoolbox", "share/qmltoolbox", &qmltoolbox::dataPath)``` instead.